source/13-transitioning/02-transitioning-to-a-new-code-of-practice.rst

```diff
@@ -21,4 +27,16 @@ New releases of the COUNTER Code of Practice will typically be assigned an effec
 * May choose to support COUNTER reports that include a range of months that span the transition period. For example, if the new reporting service compliant with a new COUNTER release was deployed in October of 2024, a customer could request a report for January-December 2024 and receive a single report in either the new release or the previous release (see previous point on the transition period).
 * When it is not practical to support a single report with date ranges that span the transition period, the report provider MUST perform the transition on the first day of a month. E.g. if the new reporting service was deployed in October 2024, a customer wanting January-December 2024 usage would request January-September 2024 from the previous reporting service and October-December 2024 from the new reporting service. For clarity, a provider MUST NOT perform the transition mid-month such that the customer is required to run reports on both the old and new reporting services for the same month and merge and sum the results to obtain actual monthly usage.
 
-Report providers MUST alert the COUNTER Project Director (tasha.mellins-cohen@counterusage.org) in advance of transitioning to a new release to ensure that the COUNTER `Registry <https://registry.projectcounter.org/>`_ remains up to date. Transition details, including the date of transition, will be noted in the specific platform record and as a notification to report consumers.
+Report providers MUST alert the COUNTER Project Director (tasha.mellins-cohen@counterusage.org) in advance of transitioning to a new release to ensure that the `COUNTER Registry <https://registry.projectcounter.org/>`_ remains up to date. Transition details, including the date of transition, will be noted in the specific platform record and as a notification to report consumers.
+
+
+New Report Providers
+""""""""""""""""""""
+
+Report providers implementing COUNTER for the first time are encouraged to implement the new release prior to the effective date of the new release. These report providers
+
+* Are not required to provide reports compliant with the new release for usage transacted prior to the implementation date, but may choose to do so at their discretion.
+* Are not required to meet the requirement to offer valid COUNTER reports for the current year plus the prior 24 months, but may choose to do so at their discretion.
+* Are not required to offer reports compliant with the prior release, but may choose to do so at their discretion.
+
+Report providers implementing COUNTER for the first time are encouraged to contact the COUNTER Project Director (tasha.mellins-cohen@counterusage.org) for advice and support and to request inclusion in the `COUNTER Registry <https://registry.projectcounter.org/>`_ per :numref:`compliance`.
```

Comment:
The reference to Section 10 seems wrong to me, it probably should point to Section 9.5?